lab-jeff/main.js

```diff
@@ -0,0 +1,5 @@
+var exports = module.exports = {};
```

Comment:
This works, but it's not a great pattern. I would go with something more like:
module.exports = exports = function cowsay(str) {do stuff};
It's also a bit less verbose